### PR TITLE
Fullcalendar3.9.0 upgrade

### PR DESCRIPTION
**Pull request by antoinepairet**

### CODE REVIEW

**Comment by antoinepairet** [comment]
@kpakur this is the same as #90 with conflicts solved and deprecated types removed
@Jamaks what do you think?

**Comment by Jamaks** [comment]
@antoinepairet thanks! I will test your mr

**Comment by Jamaks** [comment]
```
Build failed. See below for errors.

TypeError: args.indexOf is not a function
```
@antoinepairet  Merge from master latest version and pull request again please

File: src/lib/src/calendar.component.ts [hunk 1]
```diff
@@ -342,7 +342,7 @@ export class CalendarComponent implements OnInit, AfterViewInit, AfterContentChe
             });
             elem[0].dispatchEvent(widgetEvent);
         };
-        this.options.resourceRender = (resourceObj: any, labelTds: any, bodyTds: any) => {
+        (<any>this.options).resourceRender = (resourceObj: any, labelTds: any, bodyTds: any) => {
```
Comment (antoinepairet) [inline src/lib/src/calendar.component.ts]: @Jamaks This is clearly ugly
We somehow need to extend `OptionsInput` from `fullcalendar` to include the types from `fullcalendar-scheduler`

**Comment by shyamal890** [comment]
Looking forward to 3.9.0 being merged to the master. Also please consider supporting `themeStyle`

**Comment by mariohmol** [comment]
check it out https://github.com/Jamaks/ng-fullcalendar/pull/144

**Comment by antoinepairet** [comment]
Close in favor of #144

**Pull request closed without merging**